eos-tech-support/eos-download-image

```diff
@@ -25,7 +25,11 @@ def log(*args):
 
 
 def check_call(*args):
-    log('$ ' + ' '.join(map(shlex.quote, args)))
+    if shlex.quote:
+        qargs = map(shlex.quote, args)
+    else:
+        qargs = args
```

Comment:
I think you should make this `map(str, args)` in case any non string args are passed in. Otherwise you'll bomb below in the `join`. I have hit this before trying to make such a `check_call` wrapper.